softlearning/environments/adapters/robosuite_adapter.py

```diff
@@ -0,0 +1,144 @@
+"""Implements a RobosuiteAdapter that converts Robosuite envs into SoftlearningEnv."""
+
+from collections import OrderedDict
+
+import numpy as np
+import robosuite as suite
+import gym
+from gym import spaces
+
+from .softlearning_env import SoftlearningEnv
+from softlearning.environments.adapters.gym_adapter import GymAdapter
+
+ROBOSUITE_ENVIRONMENTS = {}
+
+def convert_robosuite_to_gym_obs_space(robosuite_obs_space):
+    assert isinstance(robosuite_obs_space, OrderedDict)
+    listDict = []
+    for key, value in robosuite_obs_space.items():
+        listDict.append((key, spaces.Box(
+            low=-float("inf"),
+            high=float("inf"),
+            shape=(len(value),),
+            dtype=np.float32)))
```

Comment:
Maybe use `value.dtype` here if possible?

Reply:
Yep it works!